Makefile

```diff
@@ -89,8 +90,10 @@ endif
 deps:
 	# dqlite (+raft)
 	@if [ ! -e "$(DQLITE_PATH)" ]; then \
-		git clone --depth=1 "https://github.com/canonical/dqlite" "$(DQLITE_PATH)"; \
+		echo "Retrieving dqlite from ${DQLITE_BRANCH} branch"; \
+		git clone --depth=1 --branch "${DQLITE_BRANCH}" "https://github.com/canonical/dqlite" "$(DQLITE_PATH)"; \
 	elif [ -e "$(DQLITE_PATH)/.git" ]; then \
+		echo "Updating existing dqlite branch"; \
 		cd "$(DQLITE_PATH)"; git pull; \
```

Comment:
is there a possibility that the existing dqlite clone will not be checked out to the LTS?

Reply:
yes, thats fine, i dont want to get bogged down in larger changes, and if you're working locally its nice to give devs a choice which dqlite they are using.

Reply:
indeed, good point. Thanks for clarifying :)